src/edu/csus/ecs/pc2/ui/TestResultsPane.java

```diff
@@ -1003,6 +1012,13 @@ private JPanel getResultsPaneButtonPanel() {
             // add a control button to invoke comparison of the team and judge output files for selected row(s)
             resultsPaneButtonPanel.add(getCompareSelectedButton());
 
+            //add space
+            Component horizontalStrut_4 = Box.createHorizontalStrut(20);
+            resultsPaneButtonPanel.add(horizontalStrut_4);
+            
+            // add a control button to invoke comparison of the team and judge output files for selected row(s)
```

Comment:
Comment not correct.  refers to comparison button, probably copied from above.  Should be "execute all" button?

Reply:
Updated